src/components/theme-switcher/index.tsx

```diff
@@ -58,11 +79,24 @@ export default function ThemeSelect({ id }: ThemeSelectProps) {
 	)
 }
 
-export function ThemeSelectWithLabel() {
+interface ThemeSelectWithLabelProps {
+	visuallyHideLabel?: boolean
+}
+
+export default function ThemeSelectWithLabel({
+	visuallyHideLabel = false,
+}: ThemeSelectWithLabelProps) {
 	const id = useId()
 	return (
 		<>
-			<label className={s.label} htmlFor={id}>
+			<label
+				className={classNames(
+					visuallyHideLabel ? 'g-screen-reader-only' : undefined,
+
+					s.label
+				)}
+				htmlFor={id}
+			>
```

Comment:
🙋🏻 Can we remove `visuallyHideLabel` for now since it's not used? I don't think we want to make it available for use unless absolutely necessary.

Reply:
Yeah sure!